docs/.eslintrc.js

```diff
@@ -1,29 +1,104 @@
 module.exports = {
   root: true,
-  parser: 'vue-eslint-parser',
-  plugins: ['gridsome'],
-  env: {
-    node: true,
-  },
+  plugins: ['prettier', 'node', 'jest', 'jsdoc'],
   extends: [
-    'plugin:gridsome/recommended',
-    'plugin:vue/essential',
+    'eslint:recommended',
+    'plugin:node/recommended',
     'plugin:prettier/recommended',
-    '@vue/prettier',
   ],
+  env: {
+    node: true,
+    es6: true,
+  },
+  parserOptions: {
+    ecmaVersion: 2018,
+  },
   rules: {
-    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
```

Comment:
do you have any specific reason to remove these debug lines?

Reply:
Have a look at this. https://github.com/vuejs/vue-cli/issues/2266 I think we should include this rule when this issue gets resolved.

Reply:
ok, makes perfect sense to me